srcpkgs/OpenBoardView/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'OpenBoardView'
+pkgname=OpenBoardView
+version=9.95.1
+revision=1
+create_wrksrc=yes
+build_style=cmake
+hostmakedepends="pkg-config python3"
+makedepends="SDL2-devel gtk+3-devel fontconfig-devel
+ zlib-devel"
+depends="sqlite $(vopt_if mdbtools)"
+short_desc="Open source boardview software"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="MIT"
+homepage="http://openboardview.org/"
+_obv_commit=eaf0dceb7827dc29b4c7f2073496121d1c793676
+_ghc_commit=3f1c185ab414e764c694b8171d1c4d8c5c437517
+_glad_commit=a5ca31c88a4cc5847ea012629aff3690f261c7c4
+_imgui_commit=8199457a7d9e453f8d3d9cadc14683fb54a858b5
+_mpc_commit=65f20a1a0b3249a475efa8ecb7b5ecd2c1c071c4
+_stb_commit=f54acd4e13430c5122cab4ca657705c84aa61b08
+_utf8_commit=3e9e3ec15c7bf129664ab2a113eb03b54ee0b584
+distfiles="https://github.com/OpenBoardView/OpenBoardView/archive/${_obv_commit}.tar.gz
```

Comment:
why is this using a commit instead of the version?

Reply:
Because the tar.gz package version has compilation bugs, I would have to apply many patches from the commits to compile.

https://github.com/OpenBoardView/OpenBoardView/commits/master/